movies-search-ssr/services/httpService.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const backendUrl = 'http://localhost:4000';
+
+const instanceAxios = axios.create({
+  baseURL: backendUrl,
+  timeout: 5000,
+});
+
+export default {
+  get: instanceAxios.get,
+  post: instanceAxios.post,
+  put: instanceAxios.put,
+  delete: instanceAxios.delete,
```

Comment:
I do not suggest exposing methods this way, for example, `obj.get(...)` will equal to `instanceAxios.get.call(obj, ...)` instead of `instanceAxios.get.call(instanceAxios, ...)`. The contexts are different and may result in unexpected/unknow behavior.

Reply:
if you do want to encapsulate the Axios instance, you could do it this way:

```
{
get: (...args) => instanceAxios.get(...args)
}
```